### PR TITLE
runes: hex and decimal examples

### PR DESCRIPTION
**Pull request by shakhzodkudratov**
came to this while solving logs-logs-logs's first exercise

### CODE REVIEW

**Review by junedev — COMMENTED** [review]
That's a good addition but I think the `about.md` file would be a better fit for this content. The introduction is meant for minimal information (e.g. needed to solve the exercise). We don't do a good job in keeping it minimal at the moment tbh but that is no reason to make it worse. :wink:

You can read more about the different parts of the concept here: https://exercism.org/docs/building/tracks/concepts

I had some trouble understanding the introduction sentence so I left an idea how to maybe improve it.

File: concepts/runes/introduction.md [hunk 1]
```diff
@@ -65,6 +65,15 @@ fmt.Printf("myRune Unicode code point: %U\n", myRune)
 // Output: myRune Unicode code point: U+00BF
 ```
 
+Using the table above, you can assign hexadecimal or decimal number:
```
[the remark this answers is not in the document]
Comment (shakhzodkudratov) [reply concepts/runes/introduction.md:68]: sure, sounds good!

**Comment by junedev** [comment]
@shakhzodkudratov Not sure you noticed the comment about moving the text to another file. Once that is done, I can merge this.

**Comment by shakhzodkudratov** [comment]
@junedev sorry. It is ready now.

**Review by junedev — APPROVED** [review]

**Pull request merged**